src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,218 @@
+        System.out.println(greeting);
+        System.out.println(SEPARATOR);
+    }
+
+    public static void echo(){
+        Scanner scanner = new Scanner(System.in);
+        String userInputSentence;
+
+        while (true) {
+            userInputSentence = scanner.nextLine();
+            executeCommand(userInputSentence);
+        }
+    }
+
+    private static void executeCommand(String userInputSentence) {
+        boolean isEqualToTERMIATION_SIGNAL = userInputSentence.toLowerCase().equals(Commands.TERMINATION_SIGNAL);
+        boolean isEqualToLIST_PRINT_COMMAND = userInputSentence.toLowerCase().equals(Commands.LIST_PRINT_COMMAND);
+        boolean isEqualToSET_TASK_DONE_COMMAND = userInputSentence.length() >= Commands.SET_TASK_DONE_COMMAND.length() &&
+                    userInputSentence.substring(0, Commands.SET_TASK_DONE_COMMAND.length()).equals(Commands.SET_TASK_DONE_COMMAND);
+        boolean isEqualToDELETE_COMMAND = userInputSentence.startsWith(Commands.DELETE_COMMAND);
+
+        if (isEqualToTERMIATION_SIGNAL){
+            exit();
+        } else if (isEqualToLIST_PRINT_COMMAND) {
+            printTaskList();
+        } else if (isEqualToSET_TASK_DONE_COMMAND) {
+            String[] splitCommand = userInputSentence.split(" ");
+            Integer setTaskId = Integer.valueOf(splitCommand[1]);
+
+            setTaskDone(setTaskId - 1); // -1 because array's index starts from 0
+        } else if (isEqualToDELETE_COMMAND) {
+            DeleteTask(userInputSentence);
+        } else {
+            try {
+                executeTaskAddingCommand(userInputSentence);
+            } catch (EmptyContentForTodoCommandException e) {
+                System.out.println(SEPARATOR);
+                System.out.println("☹ OOPS!!! The description of a todo cannot be empty.");
+                System.out.println(SEPARATOR);
+            } catch (UnknownCommandException e) {
+                System.out.println(SEPARATOR);
+                System.out.println("☹ OOPS!!! I'm sorry, but I don't know what that means :-(");
+                System.out.println(SEPARATOR);
+            }
+        }
+    }
+
+    private static void DeleteTask(String userInputSentence) {
+        int taskId = Integer.parseInt(userInputSentence.split(" ")[1]);
+        try {
+            Task task = tasks.get(taskId-1);
+            tasks.remove(taskId - 1);
+
+            System.out.println(SEPARATOR);
+            System.out.println("Noted. I've removed this task: ");
+            System.out.println(task);
+            System.out.printf("Now you have %d tasks in the list.\n", tasks.size());
+            System.out.println(SEPARATOR);
+        } catch (IndexOutOfBoundsException e) {
+            System.out.printf("There is no task with the index %d", taskId);
+        }
+
+        fileHandler.UpdateFile(tasks);
+    }
+
+    private static void executeTaskAddingCommand(String userInputSentence) throws EmptyContentForTodoCommandException, UnknownCommandException{
```

Comment:
Perhaps you might want to consider splitting this into multiple lines to follow coding standard?